docs/Developer-Guide.md

```diff
@@ -272,6 +272,9 @@ $ script -fec 'sudo -E USE_DOCKER=true ./image_builder.sh ${ROOTFS_DIR}'
 > - If you do *not* wish to build under Docker, remove the `USE_DOCKER`
 >   variable in the previous command and ensure the `qemu-img` command is
 >   available on your system.
+>   - If `qemu-img` is not installed, you will likely see errors such as "ERROR: File /dev/loop19p1 is not a block device"
+>     and "losetup: /tmp/tmp.bHz11oY851: Warning: file is smaller than 512 bytes; the loop device may be useless or invisible for system tools."
+>     These can be mitigated by installing `qemu-img`. For example, on Debian-based platforms, use the command `apt-get update && apt-get install -y qemu-utils`
```

Comment:
I'd consider changing the double quotes around these two error messages to backticks. That would make them stand out more but also means you don't need https://github.com/kata-containers/tests/pull/3928 as you are simply displaying a verbatim error (which arguably doesn't make sense to spell-check since it may contain errors we cannot reasonably fix).

Reply:
Ah, OK, that works for me - will add another commit now ....

Reply:
Have added a second commit @jodh-intel - does that look better ? If so, I'll close out the Issue / PR over in the tests repo

Reply:
Thanks - As mentioned, you can simplify it even more. But also, can you squash both commits into a single one?

Reply:
I *think* I've done this ... if I've made things worse, I'll just close this PR and raise a new one 👍